src/ObjectLock.php

```diff
@@ -621,11 +623,41 @@ public static function rawSearchOptionsToAdd($itemtype)
                 'massiveaction' => false,
                 'forcegroupby'  => true
             ];
+
+            $tab[] = [
+                'id'            => '207',
+                'table'         => getTableForItemType('ObjectLock'),
+                'field'         => 'id',
+                'datatype'      => 'specific',
+                'name'          => __('Lock status'),
+                'joinparams'    => ['jointype' => 'itemtype_item'],
+                'massiveaction' => false,
+                'forcegroupby'  => true,
+                'additionalfields' => ['date', 'users_id']
+            ];
         }
 
         return $tab;
     }
 
+    public static function getSpecificValueToDisplay($field, $values, array $options = [])
+    {
+        if (!is_array($values)) {
+            $values = [$field => $values];
+        }
+
+        switch ($field) {
+            case 'id':
+                return TemplateRenderer::getInstance()->render('components/search/lock_status.html.twig', [
```

Comment:
For a quick html code, I'm not sure we gain something to have dedicated template, unless you plan to reuse it.

Reply:
Always better than keeping some raw `echo` statement, no ?

Reply:
You can reuse this piece of code from #15688:

https://github.com/glpi-project/glpi/blob/0488b363681edf721ce4897964f4a8e2a9836123/src/Application/View/TemplateRenderer.php#L193-L203